### DISCUSSION TITLE
Which GPU's are suitable?

### DISCUSSION THREAD

**Discussion by Treawolfe** [lllyasviel/Fooocus#2009] [General]
Hello, I would like to experiment a bit and now wanted to run foooocus on a vm in my server (it worked very well with a 1050 ti for testing purposes) but the 1050ti is very slow. One picture - 7 to 8 minutes. I didn't want to expand the 3090 ti from my gaming computer, even though it would of course be incredibly fast.
I've searched here a bit but unfortunately couldn't find a topic, so I'm asking here now.
Which GPUs would be recommended that would offer a good compromise between performance and price?
The card doesn't have to render an image as quickly as the 3090, but it should still have decent performance. I was thinking about an RTX 4060 but unfortunately I can't find any values here in connection with fooocus. Does anyone have any suggestions or experiences which cards would be suitable and are available for a MAXIMUM of €400? The 4060 costs €300, which would of course be perfect, but I don't want to buy it only to then realize that it's not suitable and that it takes forever to finish a picture.

Sorry for my English language skills :D

**Comment by jclevet** [comment]
if this helps you in your choice:

On my RTX3070Ti,
model: mexxlLCM2_yy, guidance: 1, step: 8, sampler: dpmpp_sde_gpu
render time: 8 seconds
result is great

RTX4060 should have comparable results. Maybe 10s with the same settings.
about 30/40 seconds with default realistic settings.

**Reply by Treawolfe** [reply]
Thank you very much, I think I will actually try out a 4060 along with other reports that I have found so farThank you very much, I think I will actually try out a 4060 along with other reports that I have found so far

**Reply by mashb1t** [reply]
you will be able to generate images faster with the 4060 16GB edition though compared to the 4060 8GB edition.
It's 100$ more and i can't understand Nvidia's price policy, but just wanted to drop the information!

**Reply by piggoh** [reply]
@Treawolfe 

> Thank you very much, I think I will actually try out a 4060 along with other reports that I have found so farThank you very much, I think I will actually try out a 4060 along with other reports that I have found so far

Hi, may I ask how does 4060 perform? I am thinking to get 4060 with 16GB ram but not really sure will it work well to train a model. Thank you in advance !

**Comment by docppp** [comment]
for me (48 gb ram, rtx2060, ssd) it is about ~1.5 sec per iteration.

**Comment by wisdombox44** [comment]
Hi I have RTX 3060 12 GB, 
With the default speed for **2 images 40 second**
quality for **2 images 99 second**
extreme for 2 images 17 second
Turbo setup 
Guidance Scale 1.5
Image Sharpness 3
Sampler dpmpp_sde karras
Forced Overwrite of Sampling Step 6
**2 images 10.7 second**
*Generating and saving time: for 2 images after initial model load
RTX 3060 12 GB still okay if you are happy to wait a little, I have been using for 3 months I did not get any error messages for Fooocus
it runs very smoothly

**Comment by raochinmay6** [comment]
i order new computer see my specification and tell me fooocus will run smooth or not and yes rtx 3060 is oc
![WhatsApp Image 2024-06-01 at 17 29 57_ea765086](https://github.com/lllyasviel/Fooocus/assets/160831464/29fd354b-1fbc-4496-896c-aa20d8512965)

**Reply by mashb1t** [reply]
@raochinmay6 sure, will work, same GPU as last comment

**Reply by Rogala** [reply]
I would go for a cheaper i5 CPU and a more expensive 4000 series GPU, I don't know what your CPU needs are, maybe it needs to be that powerful.

**Comment by raochinmay6** [comment]
Please see this images and please please please please tell me how I can create same style images in fooocus please I have to create and make my own story novel 
Please help me with model settings, lora, and style 
I need to create this style images and it should be consistent 
![IMG_20240605_163003](https://github.com/lllyasviel/Fooocus/assets/160831464/8d345e26-7add-4e7a-ae3d-3f6a56e30cfa)
![IMG_20240605_162929](https://github.com/lllyasviel/Fooocus/assets/160831464/6ea4ec16-cc18-499e-a843-a605609986ae)

![IMG_20240605_162826](https://github.com/lllyasviel/Fooocus/assets/160831464/ba1856bf-0dc4-4c11-b8b1-fe2e14fc80fd)
![IMG_20240605_162729](https://github.com/lllyasviel/Fooocus/assets/160831464/9a9a0c7d-fe48-4f45-ad77-1d43d3ddcd7a)
![IMG_20240605_162638](https://github.com/lllyasviel/Fooocus/assets/160831464/cc7ea5a3-f997-4620-a236-cb39cb432a6c)

**Reply by Rogala** [reply]
You have the PC and the environment to get results, spend some time experimenting, civitai.com will help you.
